src/app/code/community/Zendesk/Zendesk/Helper/Data.php

```diff
@@ -188,4 +268,230 @@ public function isExternalIdEnabled()
     {
         return Mage::getStoreConfig('zendesk/general/use_external_id');
     }
+    
+    /**
+     * Retrieve ticket statistics
+     */
+    public function getTicketTotals($type = null, $from = null, $to = null)
+    {
+        try {
+        $tickets = Mage::getModel('zendesk/api_tickets')->all();
+        }
+        catch(Exception $ex) {
+            return array();
+        }
+        
+        if( is_null($tickets) )
+        {
+            return false;
+        }
+        
+        $totals = array(
+            'open'      =>  0,
+            'new'       =>  0,
+            'solved'    =>  0,
+            'closed'    =>  0,
+            'all'       =>  0,
+            'pending'   =>  0
+        );
+
+        if( $from )
+            $from = strtotime($from);
+        else
+            $from = 0;
+
+        foreach( $tickets as $ticket )
```

Comment:
I don't want stats calculation to be completed in the Magento extension. Stats should be calculated in the Zendesk system, then just displayed here. Unfortunately there isn't a V2 stats API so this functionality will just have to be removed for now.